client/my-sites/customer-home/cards/education/earn/index.jsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import EducationalContent from '../educational-content';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+/**
+ * Image dependencies
+ */
+import earnCardPrompt from 'assets/images/customer-home/illustration--business.svg';
+
+const EducationEarn = ( { siteSlug } ) => {
+	const translate = useTranslate();
+
+	return (
+		<EducationalContent
```

Comment:
@mmtr @kwight hmm I wonder if we should add a general tracking event for educational content cards and deprecate the old ones.

We can pass through the links as a prop on the event.